seoul_11_hongyungi/BoJ/Platinum V/Baekjoon_3015.java

```diff
@@ -0,0 +1,45 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Stack;
+
+public class Baekjoon_3015 {
+
+    static class Man {
+        int duplicate;
+        int height;
+
+        Man(int duplicate, int height) {
+            this.duplicate = duplicate;
+            this.height = height;
+        }
+    }
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        int N = Integer.parseInt(br.readLine());
+
+        Stack<Man> stack = new Stack<>();
```

Comment:
```suggestion
        Deque<Man> stack = new ArrayDeque<>();
```

태희쌤이 덱 쓰라고 하셨는데 왜 안쓰세요... 이릅니다?